src/Core/StreamableBody.php

```diff
@@ -43,24 +43,23 @@ public function getContentAsString(): string
     {
         $resource = $this->getContentAsResource();
 
-        return \stream_get_contents($resource);
+        try {
+            return \stream_get_contents($resource);
+        } finally {
+            \fclose($resource);
```

Comment:
good catch

Reply:
Yeah. I guess it is a good idea to have tests after all =)